MUXSDKImaKalturaListener/Sources/Services/MockedDispatcher.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  MockedDispatcher.swift
+//  Mux-Stats-Google-IMA-Kaltura
+//
+//  Created by Stephanie Zuñiga on 31/1/22.
+//
+
+import Foundation
+import MuxCore
+
+class MockedDispatcher: MUXSDKImaKalturaDispatcher {
+    static let shared = MockedDispatcher()
+    
+    var dispatchedEvents = [MUXSDKPlaybackEvent]()
+    
+    func dispatch(_ event: MUXSDKPlaybackEvent) {
+        print("dispatch ad event: \(event)")
```

Comment:
Is this print supposed to be here? 🤔

Reply:
Since this is the mocked dispatcher is meant to be there for testing purposes